qutip/tests/solver/heom/test_bofin_solvers.py

```diff
@@ -34,11 +34,18 @@
 )
 from qutip.solver import (
     IntegratorException,
-    SolverOptions,
-    SolverResultsOptions,
 )
 
 
+def fill_options(_=None, **kwargs):
```

Comment:
What is the `_=None` for? I'm also not sure why the `fill_options` method is here? Can't we just leave the options empty or supply the dict directly in most of the tests with less code / typing?

Reply:
When testing `Result`, the option cannot be empty since the result does not check if the options it receive is missing items. Normally the solver would ensure that they are present, but when using `Result` directly for test, we must do it manually.

I don't remember why the `_=None` is there, removing.

Reply:
Ah, I see!